Week-1/IshaanG/spec.py

```diff
@@ -0,0 +1,45 @@
+import requests
+import bs4
+import configparser
+import json
+
+
+class Spec:
+    def __init__(self):
+        self.config = self.read_config()
+
+    def smartprix_spec(self):
+        spec_all = []
+        for name in self.config['Mobile Phones'].items():
+            mobile_name = name[0]
+            list_page = requests.get(f"https://www.smartprix.com/products/?q={mobile_name}")
+            soup = bs4.BeautifulSoup(list_page.content, 'lxml')
+            phone_name = soup.find('div', class_='info').find('a').text
+            minimum_price = soup.find('span', class_='price').text
+            spec = []
+            spec.append(phone_name)
+            spec.append(minimum_price)
+            for sp in soup.find('ul', class_='pros').find_all('span'):
+                spec.append(sp.text.replace('\u2009', ''))
+            for sp in soup.find('ul', class_='cons').find_all('span'):
+                spec.append(sp.text.replace('\u2009', ''))
+            spec_all.append(spec)
+        return spec_all
+
+    def save_json(self):
+        spec_all = self.smartprix_spec()
+        data_all = []
+        for phone_spec in spec_all:
+            data = {"Name": phone_spec[0], "Price": phone_spec[1], "Connectivity": phone_spec[2], "Processor": phone_spec[3], "Memory": phone_spec[4],
```

Comment:
Line too long. I hate wrapping code and my screen is not that wide either.

https://www.python.org/dev/peps/pep-0008/#maximum-line-length